provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/task/RegisterTask.java

```diff
@@ -101,6 +102,12 @@ public void run(ResponseData responseData, Object context) throws ProvisioningDe
             throw new ProvisioningDeviceClientException(new IllegalArgumentException("authorization cannot be null"));
         }
 
+        // check if using operational CSR
+        if (provisioningDeviceClientConfig.getOperationalCertificateRequest() != null)
+        {
+            this.operationalClientCertificateRequest = provisioningDeviceClientConfig.getOperationalCertificateRequest();
+        }
+
```

Comment:
```suggestion
        // check if using operational CSR
        this.operationalClientCertificateRequest = provisioningDeviceClientConfig.getOperationalCertificateRequest();

```

There isn't any harm in assigning a null value, right?